core/include/masks/annulus2.hpp

```diff
@@ -71,6 +71,21 @@ struct annulus2 {
         std::numeric_limits<scalar>::epsilon(),
         std::numeric_limits<scalar>::epsilon()};
 
+    /** Construction from boundary values
```

Comment:
Is this just accidentally also part of this PR?

Reply:
No, I used emplace_back to fill the new store containers, because I was expecting that we build quite a few masks, so I need the right constructors for that